stories/Stepper/PalliniNavMobile.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import StepperContent from '../../src/components/Stepper/StepperContent'
+import StepperNav from '../../src/components/Stepper/StepperNav'
+import { Button, Icon, StepperContainer } from '../../src'
+import StepperDots from '../../src/components/Stepper/StepperDots'
+
+const PalliniNavMobile = () => {
+  return (
+    <StepperContainer mobile>
```

Comment:
You've missed `dark` here.

Reply:
It's not supposed to be a dark here as of BI

Reply:
https://deploy-preview-506--design-react-kit.netlify.com/?path=/story/introduzione-introduzione--benvenuto
Head over to `Navigation variations for mobile - dark version` > `Pallini` 
Match this with that of BI. You are missing a dark keyword somewhere.

Reply:
![image](https://user-images.githubusercontent.com/33497630/74681124-9ad2f300-51e8-11ea-997d-887fb27ef9cf.png)
Here's the screenshot from your PR.

Reply:
Oh, that was in `PalliniDark.js`. Will add dark right away ! 👍

Reply:
@sanyamdogra done and ready for merge !